### DISCUSSION TITLE
logstash and Original Event Forwarding

### DISCUSSION THREAD

**Discussion by elenezet** [Security-Onion-Solutions/securityonion#7556] [General]
Hi,
I try to send original event to a siem but with 2.3.110 i'have this error : 

sudo so-logstash-start
=========================================================================
Starting logstash...

This could take a while if another Salt job is running. 
Run this command with --force to stop all Salt jobs before proceeding.
=========================================================================
local:
    Data failed to compile:
----------
    Pillar failed to render with the following messages:
----------
    Rendering SLS 'logstash.manager' failed. Please see master log for details.
if I remove the custom everything works.

on a 2.3.9 with the same files it works perfectly.
 Is there a bug related to version 2.3.110?

**Comment by dougburks** [comment]
This is most likely a syntax error. Double-check your syntax.
